src/os/bluestore/NVMEDevice.cc

```diff
@@ -551,6 +551,8 @@ static bool probe_cb(void *cb_ctx, const struct spdk_nvme_transport_id *trid, st
     return false;
   }
 
+  opts->io_queue_size = UINT16_MAX;
```

Comment:
just a nit, IIUC, the `DEFAULT_IO_QUEUE_SIZE` is 256. see https://github.com/spdk/spdk/blob/62e0342eac98a8a26313e5cf9c77105d1141e01d/lib/nvme/nvme_internal.h

Reply:
Good catch @tchaikov